src/contexts/__tests__/useSdkOptions.tsx

```diff
@@ -0,0 +1,49 @@
+import { h, FunctionComponent } from 'preact'
+import { mount } from 'enzyme'
+import { EventEmitter2 } from 'eventemitter2'
+
+import { useSdkOptions, SdkOptionsProvider } from '../useSdkOptions'
+import type { NarrowSdkOptions } from '~types/commons'
+
+const DummyComponent: FunctionComponent = () => {
+  const options = useSdkOptions()
+
+  return <span data-sdk-options={options}>Options</span>
+}
+
+describe('context', () => {
+  describe('useSdkOptions', () => {
+    it('throws error when no Providers wrapped', () => {
+      expect(() => mount(<DummyComponent />)).toThrowError(
+        `SDK options wasn't initialized!`
+      )
+    })
+
+    it('gets correct options data', () => {
+      const fakeOptions: NarrowSdkOptions = {
+        token: 'fake-sdk-token',
+        containerId: 'onfido-mount',
+        language: 'fr',
+        userDetails: {
+          smsNumber: '+44765432109',
```

Comment:
Does this mobile number belong to anyone? It doesn't look like our UI test mobile number. 🤔

Reply:
Just a fake number, but now I updated it with the one we use in UI tests